envsec/internal/flow/init.go

```diff
@@ -0,0 +1,199 @@
+package flow
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/manifoldco/promptui"
+	"github.com/samber/lo"
+	"go.jetpack.io/envsec/internal/git"
+	"go.jetpack.io/pkg/api/api"
+	projectsv1alpha1 "go.jetpack.io/pkg/api/gen/priv/projects/v1alpha1"
+	"go.jetpack.io/pkg/auth/session"
+	"go.jetpack.io/pkg/id"
+	"go.jetpack.io/typeid"
+)
+
+// flow:
```

Comment:
I'm gonna leave a bunch of comments w/ suggestions for the flow, but feel free to postpone some of those to other PRs (if you agree with them). I think this is a great improvement already so let's check it in quickly and iterate on it.

Reply:
Sounds good. Yeah, I'm thinking we should iterate instead of getting it perfect. I used it a bunch and it feels pretty good (maybe 80% of where it needs to be)